app/services/member-consequences.js

```diff
@@ -148,71 +154,163 @@ export default Ember.Service.extend({
+            'Group B streptococcus (GBS)',
+            'Gestational diabetes',
+            'Hepatitis or another liver disease',
+            'Herpes',
+            'HIV/AIDS',
+            'Hypertension (high blood pressure)',
+            'Kidney disease',
+            'Lupus',
+            'Placenta previa',
+            'Preeclampsia, eclampsia, or HELLP syndrome',
+            'Rh sensitization',
+            'Rheumatoid arthritis',
+            'Syphilis',
+            'Sickle cell disease',
+            'Thalassemia',
+            'Thyroid disease',
+            'Tuberculosis',
+            'Urinary tract infections (frequent)',
+          ];
+
+          for (let i = 0; i < preg_conditions.length; i++) {
+            if (answer.contains(preg_conditions[i].dasherize())) {
+              tag.set('score', 1);
             }
           }
+          break;
 
-          if (answer.contains('condition-C')) {
-            link = consequence_links.objectAt(6);
-
-            if (! consequences.contains(link)) {
-              consequences.pushObject(link);
+        case 'problems':
+          let preg_problems = [
+            'A baby with a diagnosed genetic abnormality or birth defect',
+            'Morning sickness with significant weight loss',
+            'Cervical insufficiency (when the cervix dilates before a baby is full-term)',
+            'Premature rupture of membranes (PROM)',
+            'Preterm labor',
+            'Uterine abnormalities, such as fibroids or septate uterus',
+            'Vaginal bleeding after 12 weeks gestation',
+          ];
+
+          for (let i = 0; i < preg_problems.length; i++) {
+            if (answer.contains(preg_problems[i].dasherize())) {
+              tag.set('score', 1);
             }
           }
           break;
 
-        // Q: twins?
-        case '6':
+        case 'preterm-meds':
           if (answer.contains('yes')) {
-            link = consequence_links.objectAt(8);
+            tag.set('score', 1);
+          }
+          break;
 
-            if (! consequences.contains(link)) {
-              consequences.pushObject(link);
+        case 'smoking':
+          let smoking = [
+            'Yes, but I\'m thinking about quitting.',
+            'Yes, and I don\'t plan to quit.',
+          ];
+
+          for (let i = 0; i < smoking.length; i++) {
+            if (answer.contains(smoking[i].dasherize())) {
+              tag.set('score', 1);
+            }
+          }
+          break;
+
+        case 'drugs':
+          let drugs = [
+            'Cocaine',
+            'Heroin',
+            'Marijuana',
+            'Opioid pain medication, such as fentanyl, acetaminophen/hydrocodone, or oxycodone',
+            'Other recreational drugs',
+          ];
+
+          for (let i = 0; i < drugs.length; i++) {
+            if (answer.contains(drugs[i].dasherize())) {
+              tag.set('score', 1);
             }
           }
           break;
```

Comment:
There is a missing consequence evaluation for a question about alcohol. Check business rules and add it here.